### PR TITLE
[CLI] Review and update READMEs

### PR DESCRIPTION
**Pull request by ahsan-javaid**
## Description
This PR updates the `cli` package readme 

For details refer to issue #913

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [X] API reference/documentation update
- [ ] Other

## Does this introduce a breaking change?
No

## Are documentation updates required?
Updated `cli` readme file

## Testing information
- Copy/Paste snippets in readme should work
- Refer to `cli` readme file

## Checklist
- [ ] Code is commented where needed
- [ ] Unit test coverage for new or modified code paths
- [ ] `npm run test` passes
- [ ] Changelog is updated
- [X] Tag 1 of @yknl or @zone117x for review

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -11,3 +11,1048 @@ npm install @stacks/cli
...
+Query the balance of an account.  Returns the balances of each kind of token that the account owns.  The balances will be in the *smallest possible units* of the token (i.e. satoshis for BTC, microStacks for Stacks, etc.).
+
+Example:
+
+    $ stx balance 16pm276FpJYpm7Dv3GEaRqTVvGPTdceoY4
+    {
+      "BTC": "123456"
+      "STACKS": "123456"
+    }
+    $ stx balance SPZY1V53Z4TVRHHW9Z7SFG8CZNRAG7BD8WJ6SXD0
```
Comment (diwakergupta) [inline packages/cli/README.md]: I suggest using 1) testnet addresses unless absolutely necessary to use mainnet and 2) real addresses so ideally if someone copy / pastes any one of these example commands, it mostly works.

**Review by diwakergupta — COMMENTED** [review]
This is great, thanks @ahsan-javaid ! I imagine we can repurpose a bunch of this for docs too, cc/ @agraebe @pgray-hiro 

Meanwhile, I had one high-level comment on testnet addresses and making the examples as copy-pasteable as possible.

File: packages/cli/README.md (hunk 1 above)
Comment (ahsan-javaid) [reply packages/cli/README.md]: Added these changes. Give it a shot if it's ready for merge or need any improvements.
cc: @diwakergupta , @reedrosenbluth

File: packages/cli/README.md [hunk 2]
```diff
@@ -11,3 +11,581 @@ npm install @stacks/cli
...
+Open the terminal to try these basic commands
+
+### List of commands
+To see the usage and options for the command in general  
+```shell script
+$ stx
+``` 
+To see a list of subcommands  
+```shell script
+$ stx help
```
Comment (kyranjamie) [inline packages/cli/README.md]: I'd vote to omit the leading `$` from cli command examples. This makes it harder to use the copy/paste the commands in to your terminal, as you need to then remove the dollar sign.

Comment (ahsan-javaid) [reply packages/cli/README.md]: Added these changes.

**Review by diwakergupta — APPROVED** [review]

**Pull request merged**